cargo-dist/tests/snapshots/axolotlsay_no_locals.snap

```diff
@@ -462,14 +462,18 @@ jobs:
           # Remove the granular manifests
           rm -f artifacts/*-dist-manifest.json
       - name: Create GitHub Release
-        uses: ncipollo/release-action@v1
-        with:
-          owner: "custom-owner"
-          repo: "cool-repo"
-          token: ${{ secrets.GH_RELEASES_TOKEN }}
-          commit: ${{ fromJson(needs.host.outputs.val).ci.github.external_repo_commit }}
-          tag: ${{ needs.plan.outputs.tag }}
-          name: ${{ fromJson(needs.host.outputs.val).announcement_title }}
-          body: ${{ fromJson(needs.host.outputs.val).announcement_github_body }}
-          prerelease: ${{ fromJson(needs.host.outputs.val).announcement_is_prerelease }}
-          artifacts: "artifacts/*"
+        env:
+          PRERELEASE_FLAG: "${{ fromJson(needs.host.outputs.val).announcement_is_prerelease && '--prerelease' || '' }}"
+          ANNOUNCEMENT_TITLE: "${{ fromJson(needs.host.outputs.val).announcement_title }}"
+          ANNOUNCEMENT_BODY: "${{ fromJson(needs.host.outputs.val).announcement_github_body }}"
+          REPO: "custom-owner/cool-repo"
+          EXTERNAL_REPO_COMMIT: "${{ fromJson(needs.host.outputs.val).ci.github.external_repo_commit }}"
```

Comment:
I think that's correct in this case. The `no_locals` test has `github-releases-repo` but doesn't have the config value that defines the external commit value. In the previous version, we always passed `commit` if `github_releases_repo` was defined and relied on the fact that a null `commit:` being passed had no effect.

Reply:
So we have no tests that actually use external repo commit?

Reply:
Doesn't look like it. I'll add it to one of our existing tests.

Reply:
Oh right, adding it will actually get the build to try to read from a real submodule at that path. So if we're going to add this we'll need a repo that actually contains a submodule at the path we want it